lib/awesome_print/formatters/hash_formatter.rb

```diff
@@ -42,6 +46,19 @@ def format
 
       private
 
+      def symbol?(k)
+        k.first == ':'
```

Comment:
Oops I didn't spot this at first, but this will fail if run outside of a Rails context as String#first doesn't exist in plain old ruby™.

The original commit used `k[0]` so I'll switch back to that. I must not have run outside of Rails for quite a while and so I didn't spot the bug till now.

https://github.com/awesome-print/awesome_print/pull/267